sycl/source/detail/jit_compiler.cpp

```diff
@@ -853,21 +901,21 @@ jit_compiler::fuseKernels(QueueImplPtr Queue,
 
   static size_t FusedKernelNameIndex = 0;
   auto FusedKernelName = "fused_" + std::to_string(FusedKernelNameIndex++);
-  ::jit_compiler::KernelFusion::resetConfiguration();
+  reinterpret_cast<ResetConfigFuncT>(ResetConfigHandle)();
```

Comment:
Would it be possible to store the function pointer as `ResetConfigFuncT` in the first place to avoid the cast here?